packages/alert/index.js

```diff
@@ -0,0 +1,103 @@
+import { css, html } from 'lit';
+import { FabricElement } from '../utils';
+import { infoSvg, negativeSvg, positiveSvg, warningSvg } from './svgs';
+
+class FabricAlert extends FabricElement {
+  static properties = {
+    negative: { type: Boolean },
+    positive: { type: Boolean },
+    warning: { type: Boolean },
+    info: { type: Boolean },
+    show: { type: Boolean, reflect: true },
+    role: { type: String, reflect: true },
+  };
+
+  constructor() {
+    super();
+    this.negative = false;
+    this.positive = false;
+    this.warning = false;
+    this.info = false;
+    this.show = false;
+    this.role = 'alert';
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    if (!this.negative && !this.positive && !this.warning && !this.info) {
```

Comment:
Do we need this kind of check? The other frameworks don't do it and IMO it's fine if there's a no-op if the user is dumb.

Reply:
In React Alert's type is a required property ([source](https://github.com/fabric-ds/react/blob/d21f69a8379cc6fd4a3c49b82f1b5a7df2242847/packages/alert/src/props.tsx#L9)). 
What is the default variant in Vue then?

Reply:
There isn't one, if the user is dumb enough to not give one then bad things happen. 🤷

Reply:
Same happens here, i.e. the the alert is missing an icon and there are no colors applied. I thought that it would help a developer understand what went wrong if in addition to that weird behaviour there was an error message suggesting what went wrong and how to fix it. What do you think?

Reply:
I am 100% OK with this staying if you'd like it to. My opinion is that I try to guard/throw/warn the developer if something bad might happen and they wouldn't know about it (or if it's super bad that they do something wrong ofc).

In this case, something is _obviously_ wrong, so the error/warning is "your Alert looks stupid, fix it". The component is already visually warning the developer, so we don't really need to.